src/utils/findpackage.rs

```diff
@@ -50,6 +55,78 @@ fn handle_config_package_tst() {
 static SPECIAL_PACKAGE_PATTERN: LazyLock<regex::Regex> =
     LazyLock::new(|| regex::Regex::new(r"([a-zA-Z_\d\-]+)-(\d+(\.\d+)*)").unwrap());
 
+static CMAKE_PREFIXES: LazyLock<Vec<String>> = LazyLock::new(|| {
+    if cfg!(target_os = "linux") {
```

Comment:
#[cfg(any(
    target_os = "linux",
    target_os = "android",
    target_os = "freebsd",
    target_os = "openbsd"
))]

Reply:
or, is unix not macos?